### PR TITLE
DISCO-920 - Artwork page images do not use <img> with alt text

### PR DESCRIPTION
**Pull request by kierangillen**
- Artwork images are now using the `` element and they're using the artwork's `image_alt` field as their alt text.
- https://artsyproduct.atlassian.net/browse/DISCO-920

Screen Shot 2019-05-02 at 1 14 36 PM

Screen Shot 2019-05-02 at 1 14 42 PM

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit a759d694e42bb5522f8c3c30f69277ff69b38f8d

https://deploy-preview-2365--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 3a5a7841db8a0dd3246b01b5e96a33de8fa1b65a

https://deploy-preview-2365--artsy-reaction.netlify.com

File: src/Components/v2/Lightbox/Lightbox.tsx [hunk 1]
```diff
@@ -281,15 +288,26 @@ export class Lightbox extends React.Component<LightboxProps, LightboxState> {
...
-        })
+        const {
+          props: { src },
+        } = child
+        return (
+          enabled && (
+            <Flex
+              justifyContent="center"
+              alignItems="center"
+              height="60vh"
```
Comment (ashfurrow) [inline src/Components/v2/Lightbox/Lightbox.tsx]: If the height of the parent flex container is specified here, do we also need to specify `60vh` in `StyledImage`? Or could it be `100%`?

**Comment by mzikherman** [comment]
LGTM as well! Once the correct `alt` tag is constructed should be good to go. We can give it a once over in staging to double check.

**Comment by kierangillen** [comment]
@mzikherman Using the correct alt field now.

File: src/Components/v2/Lightbox/Lightbox.tsx (hunk 1 above)
Comment (kierangillen) [reply src/Components/v2/Lightbox/Lightbox.tsx]: `100%` would be better, just updated it!

**Comment by artsyit** [comment]
:rocket: PR was released in v16.8.1 :rocket:

**Pull request merged**